network.go

```diff
@@ -0,0 +1,66 @@
+package lndclient
+
+import (
+	"errors"
+
+	"github.com/btcsuite/btcd/chaincfg"
+)
+
+// Network defines the chain that we operate on.
+type Network uint8
```

Comment:
Why not use a string type directly?

Reply:
It isn't the same abstraction, but works too. Changed.